cpp/benchmarks/copying/contiguous_split_benchmark.cu

```diff
@@ -34,10 +34,12 @@ void BM_contiguous_split_common(benchmark::State& state,
                                 int64_t bytes_total)
 {
   // generate splits
-  cudf::size_type split_stride = num_rows / num_splits;
   std::vector<cudf::size_type> splits;
-  for (int idx = 0; idx < num_rows; idx += split_stride) {
-    splits.push_back(std::min(idx + split_stride, static_cast<cudf::size_type>(num_rows)));
+  if (num_splits > 0) {
```

Comment:
Would this not properly split up a single 2 gig single partition because the number of splits is 0? Is there a case where `num_splits` is 0 but we have work to do?